rbql/rbql_csv.py

```diff
@@ -33,11 +33,14 @@ def read_user_init_code(rbql_init_source_path):
         return src.read()
 
 
-def normalize_delim(delim):
+def normalize_delim(delim, encoding):
```

Comment:
I don't think we should decode delim in this function, let's just do it directly in rbql_main.py, because we will also need to decode the query by the very same reason and it will make sense to decode them together to keep the related parts of logic nearby:
```
if encoding == 'utf-8' and not PY3:
   delim = delim.decode('utf-8')
   query = query.decode('utf-8')
```